src/app/core/layout/layout.component.ts

```diff
@@ -1,12 +1,22 @@
-import { Component, OnInit } from "@angular/core";
+import { Component, OnInit } from '@angular/core';
 
 @Component({
-  selector: "app-layout",
-  templateUrl: "./layout.component.html",
-  styleUrls: ["./layout.component.scss"]
+  selector: 'app-layout',
+  templateUrl: './layout.component.html',
+  styleUrls: ['./layout.component.scss'],
 })
 export class LayoutComponent implements OnInit {
   constructor() {}
 
+  isLightTheme = false;
+
   ngOnInit() {}
+
+  changeTheme() {
+    if (this.isLightTheme) {
```

Comment:
this is fine, but you can even shorten this to:
```suggestion
  this.isLightTheme = !this.isLightTheme
```

That's a nice shorthand if you're JUST toggling a boolean

Reply:
> this is fine, but you can even shorten this to:
> 
> That's a nice shorthand if you're JUST toggling a boolean

I actually wrote it like this first, but in my last commit changed - I have something in my mind for later, more about that on Slack